.travis.yml

```diff
@@ -22,11 +29,14 @@ matrix:
       env: COMPILER=clang++-3.7
 
 before_install:
+  - cd r-package/grf
   - sudo apt-get update -qq
   - sudo apt-get install -y libopencv-dev valgrind
+
 script:
-  - cd core
+  - cd ../../core/
   - mkdir build && cd build
   - cmake -DCMAKE_CXX_COMPILER=$COMPILER .. && make
-  - cd .. && ./build/grf exclude:[characterization]
-  - valgrind --leak-check=full --error-exitcode=1 ./build/grf exclude:[characterization]
+  - cd .. && valgrind --leak-check=full --error-exitcode=1 ./build/grf exclude:[characterization]
```

Comment:
Super small comment, I think it's more readable to have these on separate lines.